app/controllers/proposals_controller.rb

```diff
@@ -20,6 +20,10 @@ def show
     # Get activity list
     presenter = ProposalPresenter.new(@proposal)
     @activity = presenter.activity_log
+    # Does the proposer still need to sign the CLA?
```

Comment:
As this is in both here and the edit controller, it feels like it could be pushed into the `User` model?

Reply:
That was bugging me as well, I'll move it